tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist =
+    py27
+[testenv]
+commands =
+    python -m zope.testrunner --test-path=src {posargs:-vc}
+deps =
+    .[test]
+    zope.testrunner
```

Comment:
Isn't zope.testrunner already part of the .[test] extra?  You don't install it explicitly in `.travis.yml`.